linuxNew/jdk/suse/build.gradle

```diff
@@ -67,14 +68,26 @@ task packageJdkSuse {
 			from("src/main/packaging/$product/$productVersion/")
 			into("${buildDir}/generated/packaging")
 		}
-		project.exec {
-			workingDir "src/main/packaging"
-			commandLine "docker", "build",
-				"-t", "adoptium-packages-linux-jdk-suse",
-				"-f", "Dockerfile",
-				getProjectDir().absolutePath + "/src/main/packaging"
+		if (gpgKey) {
+			project.exec {
+				workingDir "src/main/packaging"
+				commandLine "docker", "build",
+					"-t", "adoptium-packages-linux-jdk-suse",
+					"--secret", "id=gpg,src=${gpgKey}",
+					"-f", "Dockerfile",
+					getProjectDir().absolutePath + "/src/main/packaging"
+			}
+		} else {
```

Comment:
Under what circumstances do we want this to succeed if no gpg key is available? At the very least I think I'd want an option to say "Abort if none is available" to avoid accidentally producing and distributing unsigned ones.

Reply:
I've left this else statement in so that people can use other methods to sign the generated RPMs. It's also used in GitHub actions because we don't want our GPG key sitting on those executors.

Reply:
Agreed - I wouldn't be in favour of removing it but having an OPTION to ensure our own pipelines will abort if it isn't in place would seem a good idea.

Reply:
(Which is along the same lines as what @karianna said in [his comment](https://github.com/adoptium/installer/pull/373#discussion_r750526430) :-) ). I'm otherwise ok with it